editor-plugins/vscode/src/extension.ts

```diff
@@ -84,6 +84,11 @@ export function run_formatter(
     let options = PythonShell.defaultOptions;
     options.scriptPath = SCRIPT_PATH;
 
+    const pythonPath = vscode.workspace
```

Comment:
Typescript strictness will pitch a fit because it doesn't know what pythonPath is.

Explicitness would have it be `const pythonPath: string | undefined`

Reply:
Understood. Fixed it.